style.css

```diff
@@ -0,0 +1,87 @@
+
+body {
+  background-image: url('./imagees/body.png');
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
+.main {
+  background-image: url('./imagees/mobile.png');
```

Comment:
:red_circle: [Required] Kindly remove this background as it hides the body background

![Peek 2022-12-01 22-09](https://user-images.githubusercontent.com/26736582/205150083-ddb9dbcb-9e2f-45ba-a472-45ec15985fc4.gif)